src/pages/components/anchor.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Anchor
+description: >
+  The Anchor component is a link used to navigate from one location to another.
+package: '@zendeskgarden/react-buttons'
+propSheets:
+  - buttons/src/elements/Anchor.tsx
+---
+
+import AnchorDefault from '../../examples/anchor/AnchorDefault.tsx';
+import AnchorDefaultCode from '!!raw-loader!../../examples/anchor/AnchorDefault.tsx';
+import AnchorExternal from '../../examples/anchor/AnchorExternal.tsx';
+import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExternal.tsx';
+import AnchorDanger from '../../examples/anchor/AnchorDanger.tsx';
+import AnchorDangerCode from '!!raw-loader!../../examples/anchor/AnchorDanger.tsx';
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To navigate from page to page</li>
+      <li>To display links inline with text</li>
+      <li>To navigate within a page</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To prompt an action or submit data, use a <a href="/components/button">Button</a>
```

Comment:
It looks like `<a>` has yet to be mapped to markdown yet. Would you like to try mapping it so that it renders a Garden `<Anchor>`?

The existing mappings can be found [here](https://github.com/zendeskgarden/website/blob/master/src/components/MarkdownProvider/index.tsx#L78).

Reply:
Serendipitous that we're doing the Anchor page and needed this. Felt only right that I take a stab at it. Please lemme know how I fared.